activerecord/test/cases/autosave_association_test.rb

```diff
@@ -18,13 +26,6 @@
 require 'models/tag'
 require 'models/tagging'
 require 'models/treasure'
-require 'models/eye'
-require 'models/electron'
-require 'models/molecule'
-require 'models/member'
-require 'models/member_detail'
-require 'models/organization'
-require 'models/guitar'
```

Comment:
No reason other than to order alphabetically for readability. Doesn't add any value to this pull request though. Want me to change the order back to how it was?